root/programs/C#/Tests/Public.Test/Public.Test.csproj.user

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <StartAction>Program</StartAction>
+    <StartProgram>C:\root\programs\C#\Tests\packages\NUnit.Runners.2.6.3\tools\nunit.exe</StartProgram>
```

Comment:
@daisukenishino 

By default, Jenkins deploys the source code downloaded from GitHub at the following location.

```
C:\Program Files\Jenkins\jobs\[job name]\workspace
```

Is this location (C:\root\...) OK?